portability-core/src/main/java/org/dataportabilityproject/serviceProviders/google/contacts/GoogleContactsService.java

```diff
@@ -1,135 +1,157 @@
   }
 
-  private static VCard convertToModel(Person person) throws IOException {
-    VCard vcard = new VCard();
+  @VisibleForTesting
+  static VCard convertPersonToModel(Person person) throws IOException {
+    VCard vCard = new VCard();
 
     /* Reluctant to set the VCard.Kind value, since a) there aren't that many type options for
     Google contacts, b) those type options are often wrong, and c) those type options aren't even
     reliably in the same place.
     Source: https://developers.google.com/people/api/rest/v1/people#personmetadata
     */
 
-    Pair<StructuredName, StructuredName[]> namesPair = convertToVcardNames(person.getNames());
-    vcard.setStructuredName(namesPair.first);
-    vcard.setStructuredNameAlt(namesPair.second);
+    Pair<StructuredName, StructuredName[]> namesPair = convertToVCardNames(person.getNames());
+    vCard.setStructuredName(namesPair.first);
+    vCard.setStructuredNameAlt(namesPair.second);
 
-    for (Telephone telephone : convertTelephoneNumbers(person.getPhoneNumbers())) {
+    /* TODO(olsona): uncomment when we want to test it
+    for (Telephone telephone : convertToVCardTelephoneNumbers(person.getPhoneNumbers())) {
       vcard.addTelephoneNumber(telephone);
     }
 
-    for (ezvcard.property.Address vcardAddress : convertToVcardAddresses(person.getAddresses())) {
+    for (ezvcard.property.Address vcardAddress : convertToVCardAddresses(person.getAddresses())) {
       vcard.addAddress(vcardAddress);
     }
 
-    for (Email vcardEmail : convertToVcardEmails(person.getEmailAddresses())) {
+    for (Email vcardEmail : convertToVCardEmails(person.getEmailAddresses())) {
       vcard.addEmail(vcardEmail);
     }
+    */
 
-    return vcard;
+    return vCard;
   }
 
-  private static List<ezvcard.property.Address> convertToVcardAddresses(
+  @VisibleForTesting
+  static List<ezvcard.property.Address> convertToVCardAddresses(
       List<com.google.api.services.people.v1.model.Address> personAddresses) {
-    List<ezvcard.property.Address> vcardAddresses = new LinkedList<>();
+    List<ezvcard.property.Address> vCardAddresses = new LinkedList<>();
     // TODO(olsona): all of this - can use Java 8 streams
 
-    return vcardAddresses;
+    return vCardAddresses;
   }
 
-  private static List<Email> convertToVcardEmails(
-      List<EmailAddress> personEmails) {
-    List<Email> vcardEmails = new LinkedList<>();
+  @VisibleForTesting
+  static List<Email> convertToVCardEmails(List<EmailAddress> personEmails) {
+    List<Email> vCardEmails = new LinkedList<>();
     for (EmailAddress personEmail : personEmails) {
-      Email vcardEmail = new Email(personEmail.getValue());
+      vCardEmails.add(convertToVCardEmail_Single(personEmail));
+    }
 
-      if (personEmail.getMetadata().getPrimary()) {
-        vcardEmail.setPref(1);
-      }
+    return vCardEmails;
+  }
 
-      vcardEmails.add(vcardEmail);
+  @VisibleForTesting
+  static Email convertToVCardEmail_Single(EmailAddress personEmail) {
```

Comment:
how about just convertToVCardEmail? bit simpler and clear it is a single email. Also the _ is more c-style than java for function names